README.md

```diff
@@ -1,2 +1,38 @@
-# aigenda
-Aigenda is an open-source plugin designed to transform OpenAI's GPT into a daily personal assistant. It serves as a main plugin that can be enhanced with subsequent plugins for specific tasks such as manage your agenda, note taking, sending emails, reading GitHub pull requests, etc...
+<p>
+  <img src="docs/logo.png" alt="Portuguese man o' war" width="50" height="50" style="display: inline;"/> <span style="font-size: 70px;">AIgenda</span>
+</p>
+AIgenda is an open-source plugin designed to transform OpenAI's GPT into a daily personal assistant.
+
+[![License](https://img.shields.io/github/license/lusitania-labs/aigenda?style=plastic)](LICENSE)
+
+## Welcome to AIgenda: Your OpenAI GPT-Powered Personal Assistant
+
+We're excited to introduce Aigenda, an open-source plugin designed to harness the power of OpenAI's GPT and transform it into a daily personal assistant. Aigenda is more than just a plugin; it's a platform that can be extended with additional plugins to perform a variety of specific tasks. Whether you need help managing your agenda, taking notes, sending emails, or even reading GitHub pull requests, Aigenda has got you covered.
+
+But that's not all. AIgenda is developed in Kotlin, a statically typed programming language that runs on the Java Virtual Machine (JVM). 
+Why Kotlin, you ask? Our goal is to attract JVM talent into the AI space, which is currently dominated by Python.
+We believe that the JVM has a lot to offer in terms of performance, scalability, and type safety, and we want to make it easier for developers familiar with the JVM to get involved in AI development.
+
+Join us on this exciting journey as we work to make AI more accessible and useful for everyone. Whether you're an AI enthusiast, a Kotlin developer, or just someone who could use a little extra help managing your daily tasks, we invite you to try out Aigenda and see the difference it can make in your life.
```

Comment:
```suggestion
Join us on this exciting journey as we work to make AI more accessible and useful for everyone. Whether you're an AI enthusiast, a Kotlin developer, or just someone who could use a little extra help managing your daily tasks, we invite you to try out AIgenda and see the difference it can make in your life.
```